Packs/ipinfo/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Ipinfo",
     "description": "Use the ipinfo.io API to get data about an IP address",
     "support": "xsoar",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
```

Comment:
I think you can bump this to a major version, i.e. 2.0.0.
If you feel it's too much, then 1.1.0 will suffice.